crates/uv-python/python/packaging/_manylinux.py

```diff
@@ -58,6 +58,7 @@ def _have_compatible_abi(executable: str, archs: Sequence[str]) -> bool:
     if "i686" in archs:
         return _is_linux_i686(executable)
     allowed_archs = {
+        "armv5te",
```

Comment:
I'm not certain that this should be changed. I don't know if armv5te has a compatible ABI.

Reply:
I think this probably _shouldn't_ be changed, since this code is vendored and it's not present in pip etc.

Can you revert this part and re-test?

Reply:
You are right, I did not check. armv5te does not support manylinux. Reverted and tested on device, uv works fine.